### DISCUSSION TITLE
New databases creation

### DISCUSSION THREAD

**Discussion by RP-Bioinfo** [KarchinLab/open-cravat#240] [Usability & Feature Requests]
Hi, I would like to update Clinvar, Clinvar_ACMG and GnomAD to the latest releases. Do you have specific scripts for the creation of these databases starting from Clinvar XML full release and GnomAD raw files? Do you have already in program to update these databases shortly?

Best Regards,

Riccardo

**Comment by kmoad** [comment]
Clinvar, and Clinvar_ACMG are updated on the first of each month. You can get the updates with `oc module update` or in the store tab of the GUI. We are working on an updated GnomADv4 annotator https://github.com/KarchinLab/open-cravat/issues/202

I can share the clinvar build scripts if you'd like to modify them?

**Comment by RP-Bioinfo** [comment]
Ok thanks. It will be really appreciated if you share me clinvar build scripts!

**Comment by kmoad** [comment]
I've uploaded the autobuilder scripts here https://github.com/KarchinLab/open-cravat-extras/tree/main/module-builders/autobuilders

There is some internal stuff in there that is used to trigger automatic builds, but the files you want are `clinvar/clinvar.py` and `clinvar_acmg/build.py`. These should download clinvar data and build the OpenCRAVAT sqlite databases. You may have to make some minor edits to get it running on your machine.

**Comment by RP-Bioinfo** [comment]
Thanks for the scripts and for the fantastic work that you're doing on open-cravat!
